presto-verifier/src/main/java/com/facebook/presto/verifier/VerifierQueryEvent.java

```diff
@@ -41,6 +41,8 @@
     private final Double controlWallTimeSecs;
 
     private final String errorMessage;
+    private final boolean checkCpu;
```

Comment:
Remove these changes to the event. Otherwise they will get logged to the event stream. Instead use the config object that's injected into the `HumanReadableEventClient`